darts/models/forecasting/forecasting_model.py

```diff
@@ -889,6 +984,25 @@ def historical_forecasts(
                     # use retrained model if `retrain` is not training every step
                     model = model if model is not None else self
 
+                    # slice the series for prediction without retraining
```

Comment:
Reformulating the two cases for eventual other reviewers:
- the model is not trained for the first "predictable" timestamp -> user should either call `fit()` before `historical_forecasts`, modify the `retrain` argument so that it returns `True` at least once before/at this timestamp or change the `start` value to the suggested timestamp so that the model training can be triggered before the prediction round.
- `retrain` is False for the first "trainable" timestamp and the model was not trained before `historical_forecasts` -> user must modify the `retrain` argument.